securedrop/source_templates/lookup.html

```diff
@@ -29,7 +29,7 @@ <h1 class="headline">{{ gettext('Submit Materials') }}</h1>
   <div class="snippet">
     <div class="attachment grid-item center">
       <img class="center" src="{{ url_for('static', filename='i/server_upload.png') }}" width="73px" height="62px">
-      <input type="file" name="fh" autocomplete="off">
+      <input type="file" name="fh[]" autocomplete="off" multiple>
```

Comment:
Currently a source can upload single files up to 500 MB in size. After your PR, a source can upload _unlimited_ files, each up to 500 MB in size. My concern here is [file space exhaustion](https://www.owasp.org/index.php/Unrestricted_File_Upload) due to the unrestricted uploads. Maybe we should limit this to 10 files or so - what do you think? Then we can add descriptive text saying "Attach up to 10 files at a time, each up to 500 MB in size", as done in this UI screenshot: http://ui-patterns.com/users/32814/collections/upload-file/screenshots/17805.

Reply:
Limiting the actual number of uploads sounds good. I was thinking to do a simple list slicing in the server side too, and have only 10 files as submissions at any given submit operation.

Reply:
Hey @kushaldas - it looks like this comment still needs to be addressed.

Reply:
More reading +testing the interface shows that [500MB size limit](https://github.com/freedomofpress/securedrop/blob/develop/install_files/ansible-base/roles/app/templates/sites-available/source.conf#L58) is per request, not on per file. Means even if the source uploads 100 files, the total size can not exceeds 500MB per request.

Also because we don't have any JS on the client side, I could not find any way to restrict the maximum number of files to be attached.

Reply:
The official Apache documentation is [available here](https://httpd.apache.org/docs/2.4/mod/core.html#limitrequestbody).